package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "preact-signals",
 	"version": "1.0.0",
-	"description": "",
+	"description": "Manage state with style in every framework",
```

Comment:
Setting this will do nothing, as this is the root `package.json` for the monorepo. It's here to help with common tooling and whatnot, it's not a package itself.

You'll instead want to set this in each package, so here in the case of `@preact/signals`:

https://github.com/preactjs/signals/blob/d3a566e1842ab93505070a403f800e3f4a999557/packages/preact/package.json#L1-L5

Reply:
Yes, sorry for the mistake.

Reply:
No need to apologize!

We'd still love your contribution, if you want to update the correct files & re-open this PR I'd be happy to merge it in

Reply:
✅